core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -2314,6 +2314,15 @@ private Sql checkSubQuery(String sql) {
     checkSubQuery(sql).check();
   }
 
+  @Test public void testDistinctNonDistinctAggregates() {
+    final HepProgram program = HepProgram.builder()
+            .addRuleInstance(AggregateExpandDistinctAggregatesRule.JOIN)
+            .build();
+    checkPlanning(program, "select emp.empno, count(*), avg(distinct dept.deptno) \n"
+            + " from sales.emp emp inner join sales.dept dept \n"
+            + " on emp.deptno = dept.deptno \n"
```

Comment:
Can you remove the spaces before the line feeds? they are unncessary, but they turn into spaces at the ends of lines in RelOptRulesTest.xml which somebody's editor will mysteriously remove at some point in the future.